### PR TITLE
Feature/form validation tests

### PR DESCRIPTION
**Pull request by jeffvg**
#### Changes proposed in this Pull Request:
- add form validation tests
- update view port test
- 

#### To Test:
-

### CODE REVIEW

**Comment by jeffvg** [comment]
I'm going to update the tests in 4 of the design patterns to verify state rather than color displayed for error

File: cypress/integration/form-validation/phone-number.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/// <reference types="cypress" />
+
+
+describe('Form validation', () => {
```
Comment (emclaug2) [inline cypress/integration/form-validation/phone-number.spec.js:4]: These describe blocks, should be they more specific than just 'form validation'? Each example should have its own unique title.

File: cypress/integration/form-validation/sectioned-form.spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        .should('contain', 'Required')
+        cy.get('#email-field-helper-text')
+        .should('contain', 'Required')
+    });
+
+    it('should display not required for fields after refresh', () => {
+        cy.get('[data-cy=submit]').click()
+        cy.get('#address-field-helper-text')
+        .should('contain', 'Required')
+        .invoke('css', 'color').should('equal', 'rgb(202, 60, 61)')
```
Comment (emclaug2) [inline cypress/integration/form-validation/sectioned-form.spec.js:35]: Check for warn prop instead of color?

**Review by emclaug2 — CHANGES_REQUESTED** [review]
Tests look very thorough, just see 2 small comments.

File: cypress/integration/form-validation/sectioned-form.spec.js (hunk 2 above)
Comment (jeffvg) [reply cypress/integration/form-validation/sectioned-form.spec.js]: removed css color check. Kept should can required as its passed from child prop

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**